tools/ubuntu_setup.sh

```diff
@@ -81,6 +83,7 @@ git submodule init
 git submodule update
 
 # install python
+PATH=$HOME/.pyenv/bin:$HOME/.pyenv/shims:$PATH
```

Comment:
I made your suggested changes without commenting anything, hope that is okey?

this was the comment i had planned for the pip3 line:
ahhh now i see, in .pyenv/shims you have a pip executable
Will edit PATH= on line 89 to :
PATH=$HOME/.pyenv/bin:$HOME/.pyenv/shims:$PATH
(so i will revert this pip3 change)

the reason for the PATH issues could be related to this warning when you run this command
```
$ eval "$(pyenv init -)"
WARNING: `pyenv init -` no longer sets PATH
Run `pyenv init` to see the necessary changes to make your configuration
```
And `which pyenv` is pointing to $HOME/.pyenv/bin/pyenv

Reply:
makes sense